src/main/scala/nl/knaw/dans/easy/stage/fileitem/FileItemConf.scala

```diff
@@ -117,22 +111,21 @@ class FileItemConf(args: Seq[String]) extends ScallopConf(args) {
 
   mainOptions = Seq(datasetId, dsLocation, pathInDataset, size, format)
 
-  dependsOnAll(format,List(datasetId,pathInDataset,size,dsLocation))
-  dependsOnAll(datasetId,List(pathInDataset,size,dsLocation))
-  conflicts(csvFile,List(datasetId,pathInDataset,size,dsLocation))
-  requireOne(csvFile,datasetId)
-  mutuallyExclusive(isMendeley, file)
+  dependsOnAll(format, List(datasetId, pathInDataset, size, dsLocation))
+  dependsOnAll(datasetId, List(pathInDataset, size, dsLocation))
+  conflicts(csvFile, List(datasetId, pathInDataset, size, dsLocation))
+  requireOne(csvFile, datasetId)
 
   validate(creatorRole) (s => validateValue(s, creatorRoles))
 
-  val longOptionNames = builder.opts.filter(!_.isInstanceOf[TrailingArgsOption]).map(_.name)
+  val longOptionNames = builder.opts.filter(o => !o.isInstanceOf[TrailingArgsOption] && o.name != "csv-file").map(_.name)
```

Comment:
You could have added an extra `filter` as well 😄 
`.filter(predicate1 && predicate2)` is equal to `.filter(predicate1).filter(predicate2)`

*just for info*